src/components/seo.tsx

```diff
@@ -51,14 +52,39 @@ const SEO: React.FC<SeoProps> = ({
   let pathname;
   if (typeof window !== 'undefined') {
```

Comment:
We cannot use this check in the render function. This will lead to a different page that gets rendered on SSR and in the browser (important initial render). We also need to make sure the set the headers independently of the browser, to have them in place on the very first request.

The translations need to be passed into the SEO component directly from Gatsby. This array should then be used to render the correct headers.